### PR TITLE
Add support for Pensando DSC card

### PR DESCRIPTION
**Pull request by vrindle**

### CODE REVIEW

File: bundle/manifests/supported-nic-ids_v1_configmap.yaml [hunk 1]
```diff
@@ -25,6 +25,8 @@ data:
   Qlogic_qede_QL41000: 1077 8070 8090
   Qlogic_qede_QL45000_25G: 1077 1656 1664
   Qlogic_qede_QL45000_50G: 1077 1654 1664
+  Pensando_DSC_25GB: "1dd8 1002 1003"
+  Pensando_DSC_100GB: "1dd8 1002 1003"
```
Comment (zshi-redhat) [inline bundle/manifests/supported-nic-ids_v1_configmap.yaml:29]: Remove these two lines as we don't use them in openshift deployment.

File: config/manifests/bases/sriov-network-operator_configmap.yaml [hunk 2]
```diff
@@ -28,3 +28,5 @@ data:
   Qlogic_qede_QL41000: "1077 8070 8090"
   Qlogic_qede_QL45000_25G: "1077 1656 1664"
   Qlogic_qede_QL45000_50G: "1077 1654 1664"
+  Pensando_DSC_25GB: "1dd8 1002 1003"
+  Pensando_DSC_100GB: "1dd8 1002 1003"
```
Comment (zshi-redhat) [inline config/manifests/bases/sriov-network-operator_configmap.yaml:32]: Remove these two lines as we don't use them in openshift deployment

File: deploy/configmap.yaml [hunk 3]
```diff
@@ -21,4 +21,6 @@ data:
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
   Qlogic_qede_QL45000_50G: "1077 1654 1664"
-  
+  Pensando_DSC_25GB: "1dd8 1002 1003"
+  Pensando_DSC_100GB: "1dd8 1002 1003"
```
Comment (zshi-redhat) [inline deploy/configmap.yaml:25]: Remove these two lines as we don't use them in openshift deployment

File: deployment/sriov-network-operator/templates/configmap.yaml [hunk 4]
```diff
@@ -21,4 +21,6 @@ data:
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
   Qlogic_qede_QL45000_50G: "1077 1654 1664"
+  Pensando_DSC_25GB: "1dd8 1002 1003"
+  Pensando_DSC_100GB: "1dd8 1002 1003"
```
Comment (zshi-redhat) [inline deployment/sriov-network-operator/templates/configmap.yaml:25]: Remove these two lines as we don't use them in openshift deployment

File: manifests/stable/supported-nic-ids_v1_configmap.yaml [hunk 5]
```diff
@@ -27,6 +27,8 @@ data:
   Qlogic_qede_QL41000: 1077 8070 8090
   Qlogic_qede_QL45000_25G: 1077 1656 1664
   Qlogic_qede_QL45000_50G: 1077 1654 1664
+  Pensando_DSC_25GB: "1dd8 1002 1003"
+  Pensando_DSC_100GB: "1dd8 1002 1003"
```
Comment (zshi-redhat) [inline manifests/stable/supported-nic-ids_v1_configmap.yaml:31]: Combine 25G and 100G IDs to single line because they are the same.

**Review by zshi-redhat — CHANGES_REQUESTED** [review]

File: manifests/stable/supported-nic-ids_v1_configmap.yaml [hunk 6]
```diff
@@ -27,6 +27,7 @@ data:
   Qlogic_qede_QL41000: 1077 8070 8090
   Qlogic_qede_QL45000_25G: 1077 1656 1664
   Qlogic_qede_QL45000_50G: 1077 1654 1664
+  Pensando_DSC_25GB_100GB: "1dd8 1002 1003"
```
Comment (zshi-redhat) [inline manifests/stable/supported-nic-ids_v1_configmap.yaml:30]: Could you remove the quotes?

Comment (vrindle) [reply manifests/stable/supported-nic-ids_v1_configmap.yaml:30]: It seems support for pensando card has already been added to this file. 
See: [https://github.com/openshift/sriov-network-operator/blob/d9a8cbb989c0dcd7decf28a25bc26834336fa5eb/manifests/stable/supported-nic-ids_v1_configmap.yaml#L26](url)

I will just remove this line altogether

**Review by zshi-redhat — APPROVED** [review]

**Comment by SchSeba** [comment]
@vrindle can you open this PR to the u/s repo please?
https://github.com/k8snetworkplumbingwg/sriov-network-operator

**Comment by zshi-redhat** [comment]
> @vrindle can you open this PR to the u/s repo please? https://github.com/k8snetworkplumbingwg/sriov-network-operator

I just realized this change is not necessary, since the supported ID only needs to go to the manifests folder (openshift specific).
And this PR currently is downstream only, which may cause merge conflict for futher backport. 
Maybe we shall just revert this change, because adding it to upstream means us need to maitain it which is not what we want now.

**Pull request merged**